Tests/Functional/ServiceInstantiationTest.php

```diff
@@ -71,4 +76,21 @@ public function testProfilingDecoration()
         $this->assertInstanceOf(ProfilePlugin::class, $plugins[3]);
         $this->assertInstanceOf(ProfilePlugin::class, $plugins[4]);
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected static function bootKernel(array $options = [])
+    {
+        parent::bootKernel($options);
+
+        /** @var EventDispatcherInterface $dispatcher */
+        $dispatcher = static::$kernel->getContainer()->get('event_dispatcher');
+
+        $event = new GetResponseEvent(static::$kernel, new Request(), HttpKernelInterface::MASTER_REQUEST);
+
+        $dispatcher->dispatch(KernelEvents::REQUEST, $event);
+
+        return static::$kernel;
```

Comment:
this might be confusing because the base test cases don't do it in older symfony versions. but i think its still a good idea. assigning the bootKernel return value, then accessing static::$kernel and in the end return the variable would be weird.